Jenkinsfile

```diff
@@ -99,7 +99,7 @@ pipeline {
 def dockerBuild(version) {
     // dockerhub is the ID of the credentials stored in Jenkins
     docker.withRegistry('https://index.docker.io/v1/', 'dockerhub') {
-        git poll: false, url: 'https://github.com/AdoptOpenJDK/openjdk-docker.git'
+        git poll: false, url: 'https://github.com/sxa/openjdk-docker.git', branch: 'purge_cache'
```

Comment:
should that be Adoptium?

Reply:
No - this is the container pushes for the adoptopenjdk dockerhub registry so it should be left as AdoptOpenJDK (we agreed to keep this active in a PMC meeting a couple of months back to keep them current for existing users who haven't yet updated) as we generally wouldn't want to have AdoptOpenJDK-badged stuff under github.com/adoptium. I'll need to adjust this after it's been approved, but at the moment if it's set to that in order to pull from my branch to be able to test it.

I've reverted this line so it's correct, but it means that the problem resolved by this PR will now recur unless this is merged quickly. Hopefully we can get it in today.